torch/csrc/jit/script/compiler.h

```diff
@@ -130,7 +130,7 @@ struct TORCH_API BuiltinFunction : public SugaredValue {
 
 struct TORCH_API BuiltinModule : public SugaredValue {
   BuiltinModule(const std::string& name)
-    : name(name) {}
+    : name(std::move(name)) {}
```

Comment:
You're moving a const reference???

Reply:
I forgot to change `const std::string& name` to `std::string name`. Will fix